src/components/SidePanels/DetailsPanel/DetailsPanel.styl

```diff
@@ -22,11 +22,14 @@
         font-weight normal
 
 .region-actions
-  margin 8px
-  height 36px
+  padding 8px
+  height 52px
   display flex
   align-items stretch
   justify-content space-between
+  position sticky
```

Comment:
comment from @Gondragos: small usual regions now have a bit weird floating appearance of actions panel on scroll. possibly can be fixed with some `top: 50vh` to trigger it only on huge content regions

Reply:
but this is only actual for multiple selected regions when the overall height is higher than screen. and it's not critical anyway